client/js/components/GameView/TargetSelectionOverlay.vue

```diff
@@ -15,7 +15,16 @@
       <v-btn outlined color="primary" class="mt-2" data-cy="cancel-target" @click="$emit('cancel')">
         Cancel
       </v-btn>
+      
     </div>
+    <v-btn
+        color="primary"
+        text
+        data-cy="cancel-target-mobile"
+        @click="$emit('cancel')"
+      >
+        X
+    </v-btn>
```

Comment:
This works, but let's make it more consistent with the X button in the `MoveChoiceOverlay` by making it an icon button and using the `mdi-close` icon instead of the letter X. Let's also make the icon extra large:

```
      <v-btn icon data-cy="cancel-target-mobile" @click="$emit('cancel')">
        <v-icon x-large>
          mdi-close
        </v-icon>
      </v-btn>
```